docs/workflow-make-repo.md

```diff
@@ -9,7 +9,7 @@ Click the green button that says <span style="background-color: green; color: wh
 Fill in the details:
 
 - **owner**: select your personal GitHub for testing/experimenting, or select the opensafely organisation for a bona fide OpenSAFELY-approved study. The repo can be transferred into opensafely later if needed.
-- **repository name**: This should always end in `-research` so that any back-end updates to the template can be pushed through to existing repos. The rest of the name should be something short but informative --- browse existing repo names for inspiration.
+- **repository name**: The name should be short but informative &mdash; browse existing repo names for inspiration.
```

Comment:
Last I knew this was still a convention. I'd like to change that convention/soft requirements, but I think we need a chat with tech team first. Possibly I missed a conversation here in which case sorry!

Reply:
from https://ebmdatalab.slack.com/archives/C63UXGB8E/p1607357521090500?thread_ts=1607357374.088700&cid=C63UXGB8E

>  we have an explicit list of non-research repos, everything else is assumed research

but let me know if you still want me to revert

Reply:
So, we don't *rely* on the convention for security purposes, but that doesn't mean we shouldn't still prefer a convention.

Reply:
ok have put back in